cumulusci/tasks/bulkdata/mapping_parser.py

```diff
@@ -98,6 +99,7 @@ class MappingStep(CCIDictModel):
     ] = None  # default should come from task options
     anchor_date: Optional[Union[str, date]] = None
     soql_filter: Optional[str] = None  # soql_filter property
+    update_key: str = None  # only for upserts
```

Comment:
When you add the synthetic upsert feature with support for compound keys, is the intent to keep the singular name `update_key` but let it also accept a list?

Reply:
Good question and yes, I think that a "compound key" works semantically.